### DISCUSSION TITLE
Отключение глобального OOC чата

### DISCUSSION THREAD

**Discussion by pr0pk1** [octothorp-team/public-dobrograd#751] [Предложения]
Че я предлогаю, так это в настройках добавить пункт - "Отключение сообщения о приезде игрока", ибо очень часто мешает. Также, еще одна функция - "Отключение глобального OOC чата", уже понять можно что две приколы делают.

**Comment by KillerBunny031** [comment]
Ну так это, ты хоть смотри что есть, а чего нету, когда предложение создаешь. Отключение "приезда игрока в город" в настройках уже есть. А отключать ООС не вижу смысла, ведь иногда он нужен для того, чтобы разобраться в ситах.
